apps/src/maze/beeBlocks.js

```diff
@@ -4,6 +4,8 @@
 
 var msg = require('./locale');
 var blockUtils = require('../block_utils');
+var BlockStyles = require('../blockly/constants').BlockStyles;
```

Comment:
can we use more modern import syntax here?

Reply:
I was worried about potential implications of changing the existing `require` statements to imports, so I thought for now I would just match what I saw rather than try to update the whole file. Do you think it would be straight forward to just convert all requires to imports in these files?

Reply:
I _think_ it would be straightforward (worth testing to make sure). Also fine to leave it as-is to keep with the current pattern.